### PR TITLE
Restore coach filters

### LINKED ISSUE #9886: Update messages for filters on CoachExamsPage (and lessons too)
I spotted this while reviewing #9863 and saw some linting changes in the diff for `CoachExamsPage` where there are comments indicating that we were temporarily commenting out the Active/Inactive filters for exams in #6174 due to not having updated the strings.

With the guidance from @radinamatic in #6174 we need to be sure we have the correct strings defined in the component and that the select dropdown is visible and working as expected.

Note that `CoachExamsPage` isn't the only place where this is an issue, there are others as can be seen in https://github.com/learningequality/kolibri/pull/6233/files (the PR that commented the filters out).

### PR DESCRIPTION
**Pull request by akolson**
## Summary

This PR restores coach filters disabled in https://github.com/learningequality/kolibri/issues/6174. Also, based on feedback from @jtamiace an additional filter option **Ended** has been added to the quizes filter and appropriate filtering logic added.

Screenshot 2022-12-21 at 19 06 36
Screenshot 2022-12-21 at 19 07 06
Screenshot 2022-12-21 at 19 08 24
Screenshot 2022-12-21 at 19 09 33

## References

Closed #9886 

## Reviewer guidance

- Use the lessons and quiz filters under Coach Reports and Plan tabs as shown in the screenshots attached

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Review by nucleogenesis — APPROVED** [review]
Code LGTM! I'd wait for @radinamatic 's review on the strings before merging.

**Comment by pcenov** [comment]
@akolson and @radinamatic LGTM both filters are reinstated and are functioning correctly as specified above. Note that the strings are not localized though.

File: kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js [hunk 1]
```diff
@@ -56,11 +56,11 @@ const coachStrings = createTranslator('CommonCoachStrings', {
   },
 
   // labels, phrases, titles, headers...
-  // activeLabel: 'Active',
-  // activeQuizzesLabel: {
-  //   message: 'Active quizzes',
-  //   context: 'An active quiz is one that is in progress.',
-  // },
+  activeLabel: 'Active',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js]: I'm confused, why are these strings re-activated? There is no concept of **Active/Inactive** any more, neither for lessons nor for quizzes... 😕 

Quizzes can be **Not started**, **Started** and **Ended**.
Lessons can be **Visible** and **Not visible** (to learners).

We should delete all the instances of _Active/Inactive_ strings when they are supposed to refer to lessons or quizzes.

File: kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js [hunk 2]
```diff
@@ -56,11 +56,11 @@ const coachStrings = createTranslator('CommonCoachStrings', {
...
+  activeQuizzesLabel: {
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js [hunk 3]
```diff
@@ -151,12 +196,12 @@ const coachStrings = createTranslator('CommonCoachStrings', {
     context:
       "In the 'Difficult questions' sub-tab within the 'Reports' section, there's a column called 'Help needed' which shows the coach which learners need help on what questions.\n\nCoaches can also filter class activity by 'Help needed'",
   },
-  // inactiveQuizzesLabel: 'Inactive quizzes',
+  inactiveQuizzesLabel: 'Inactive quizzes',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js [hunk 4]
```diff
@@ -151,12 +196,12 @@ const coachStrings = createTranslator('CommonCoachStrings', {
...
   lastActivityLabel: {
     message: 'Last activity',
     context: 'Indicates when a learner was last active on a particular resource.',
   },
-  // inactiveLabel: 'Inactive',
+  inactiveLabel: 'Inactive',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue [hunk 5]
```diff
@@ -108,18 +106,24 @@
...
-          v-else-if=" statusSelected.value === coachString('inactiveQuizzesLabel') &&
-            !inactiveExams.length"
-        >
-          {{ $tr('noInactiveExams') }}
-        </p> -->
+      <p
+        v-else-if="statusSelected.value === coachString('filterQuizStarted') &&
+          !activeExams.length"
+      >
+        {{ $tr('noActiveExams') }}
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue]: Not sure what exactly is this condition supposed to do, but filtering by _active exams_ makes no sense any more...

File: kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue [hunk 6]
```diff
@@ -108,18 +106,24 @@
...
+      </p>
+      <p
+        v-else-if=" statusSelected.value === coachString('filterQuizNotStarted') &&
+          !inactiveExams.length"
+      >
+        {{ $tr('noInactiveExams') }}
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue]: Same as above for _active exams_.

File: kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue [hunk 7]
```diff
@@ -283,14 +293,17 @@
...
       },
-      // noActiveExams: 'No active quizzes',
-      // noInactiveExams: {
-      //   message: 'No inactive quizzes',
-      //   context:
-      //     "Inactive quizzes are ones that are no longer in progress.
-      //     When the coach presses the 'End quiz' button, the quiz
-      //passes from 'active' to 'inactive'.",
-      // },
+      noActiveExams: 'No active quizzes',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue [hunk 8]
```diff
@@ -283,14 +293,17 @@
...
+      noEndedExams: {
+        message: 'No ended quizzes',
+        context:
+          'Message displayed when there are no ended quizes. Ended quizzes are those that are no longer in progress.',
+      },
+      noInactiveExams: {
+        message: 'No inactive quizzes',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/plan/LessonsRootPage/index.vue [hunk 9]
```diff
@@ -241,8 +240,8 @@
         context:
           "Text displayed in the 'Lessons' tab of the 'Plan' section if there are no lessons created",
       },
-      // noActiveLessons: 'No active lessons',
-      // noInactiveLessons: 'No inactive lessons',
+      noActiveLessons: 'No active lessons',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/plan/LessonsRootPage/index.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/plan/LessonsRootPage/index.vue [hunk 10]
```diff
@@ -241,8 +240,8 @@
...
+      noInactiveLessons: 'No inactive lessons',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/plan/LessonsRootPage/index.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue [hunk 11]
```diff
@@ -103,29 +102,29 @@
         if (this.filter.value === 'allLessons') {
           return this.coachString('lessonListEmptyState');
         }
-        // if (this.filter.value === 'activeLessons') {
-        //   return this.$tr('noActiveLessons');
-        // }
-        // if (this.filter.value === 'inactiveLessons') {
-        //   return this.$tr('noInactiveLessons');
-        // }
+        if (this.filter.value === 'activeLessons') {
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue]: There should be no filters for active/inactive lessons or quizzes.

File: kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue [hunk 12]
```diff
@@ -188,10 +187,10 @@
       },
     },
     $trs: {
-      // activeLessons: 'Active lessons',
-      // inactiveLessons: 'Inactive lessons',
-      // noActiveLessons: 'No active lessons',
-      // noInactiveLessons: 'No inactive lessons',
+      activeLessons: 'Active lessons',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue [hunk 13]
```diff
@@ -188,10 +187,10 @@
...
+      inactiveLessons: 'Inactive lessons',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue [hunk 14]
```diff
@@ -188,10 +187,10 @@
...
+      noActiveLessons: 'No active lessons',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue [hunk 15]
```diff
@@ -188,10 +187,10 @@
...
+      noInactiveLessons: 'No inactive lessons',
```
Comment (radinamatic) [inline kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue]: Delete the string, not needed.

File: kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue (hunk 5 above)
Comment (akolson) [reply kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue]: @radinamatic  These are specific messages displayed if no exams are returned on filtering. The confusion arises(apologies on the confusion!) with the names of the translation strings. These have been refactors to reflect started, not started and ended.

File: kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue (hunk 15 above)
Comment (akolson) [reply kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue]: @radinamatic `printLabel` string seems to be used for file names during exports(Ln 184) + also as a report header(Ln 10)

File: kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue (hunk 11 above)
Comment (akolson) [reply kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue]: Apologies on the confusion, the tranlation strings have been refactored to visible and not visible. This are messages displayed incase the filter returns no rows.

**Comment by akolson** [comment]
@radinamatic translation string changes have been effected and pr is ready for a re-review. Thanks

**Review by marcellamaki — APPROVED** [review]
Final updates look good, and manual re-review after last adjustments check out.

**Pull request merged**